Scripts/RemediationScripts/Remediate-DontUseNonADIdentities.ps1

```diff
@@ -352,22 +385,27 @@ function Remove-AzTSNonADIdentities
         Write-Host "Path: $($folderpath)NonADAccountsRoleAssignments.json"
     }
 
-    if(-not $Force)
-    {
-        Write-Host "Do you want to delete the above listed role assignment? " -ForegroundColor Yellow -NoNewline
-        $UserInput = Read-Host -Prompt "(Y|N)"
+    # Safe Check: Exporting all role assignments in CSV file.
+    $externalAccountsRoleAssignments | Export-CSV -Path "$($folderpath)\NonADAccountsRoleAssignments.csv" -NoTypeInformation
```

Comment:
Do you want to terminate the script (for this as well as the above case), when there are no role assignments present?

Reply:
If there is no role assignment then script will get terminated before this.